front/src/reducers/osrdconf/types.ts

```diff
@@ -151,16 +151,9 @@ export type PathStepMetadata =
        */
       trackName?: string;
       /**
-       * An OperationalPoint is not unique by its UIC but by its UIC + secondary code.
-       *
-       * However, it can contains multiple parts since it can be referenced on multiple tracks.
-       *
-       * This Map stores all possible locations for a given op's secondary code.
+       * Data of all parts (tracks) of the path step (name + ch)
        */
-      locationsBySecondaryCode: Map<
-        string,
-        { trackId: string; trackName: string; lineName: string; coordinates: Position }[]
-      >;
+      parts: { id: string; name: string; coordinates: Position }[];
```

Comment:
id is the track_id right ? or maybe we should rename `parts` by `tracks` ?
```suggestion
      parts: { trackId: string; name: string; coordinates: Position }[];
```

Reply:
Yes that's right, I chose this name for consistency with `OperationalPoint` type but with just `id` and `name` maybe it's hard to follow what we are talking about.

Reply:
Changed the field with `trackId` and `trackName`